libs/interpret-text/src/lib/TextExplanationDashboard/Control/TextHighlighting/TextHightlighting.tsx

```diff
@@ -47,37 +45,30 @@ export class TextHighlighting extends React.PureComponent<IChartProps> {
             styles={textStackStyles}
           >
             {text.map((word, wordIndex) => {
+              const isWordSelected =
+                (this.props.selectedTokenIndex &&
+                  wordIndex === this.props.selectedTokenIndex) ||
+                false;
```

Comment:
nit: I think you can remove the || false part? just
```
const isWordSelected = this.props.selectedTokenIndex && wordIndex === this.props.selectedTokenIndex
```

Reply:
It results in isWordSelected being boolean | 0 | undefined which is not acceptable in textHighlightingStyles